### DISCUSSION TITLE
Docker Compose guide seems outdated?

### DISCUSSION THREAD

**Discussion by caschoener** [fief-dev/fief#355] [Q&A]
https://docs.fief.dev/self-hosting/deployment/docker-compose/

The guide suggests running fief run-server and fief run-worker in my docker images.  With the latest code, I just get errors `No such command 'run-worker'.` and `No such command 'run-server'.`

I tried with 0.27 and it works fine there.

**Comment by frankie567** [comment]
Starting 0.28+, we're doing a trick with the CLI to avoid problems when environment variables are missing or wrong: the main commands are not loaded; hence the `No such command` message.

Check if you have correctly configured all the mandatory environment variables:

* SECRET
* FIEF_CLIENT_ID
* FIEF_CLIENT_SECRET
* ENCRYPTION_KEY
* FIEF_DOMAIN

---

I'll open an issue to improve the CLI so it outputs meaningful info when we are in this case.
